src/main/java/wellnus/atomichabit/command/UpdateCommand.java

```diff
@@ -133,16 +156,15 @@ protected String getSupportedCommandArguments() {
 
     /**
      * Executes the update command for atomic habits.<br>
-     *
+     * <p>
      * This command is interactive, so user will continue providing arguments via
-     *     further prompts provided.
+     * further prompts provided.
```

Comment:
Hmm i wouldn't remove this indentation, it makes it clear that this is a single paragraph

Reply:
Nevermind this is wrong, the correct way is no indentation!